catalog/transformer/catalog-transformer-streaming-lib/src/test/java/org/codice/ddf/transformer/xml/streaming/lib/TestSaxEventToXmlElementConverter.java

```diff
@@ -33,27 +33,51 @@
      */
 
     @Test
-    public void testSaxEventToXmlElementConverterRedeclaredNamespaceUri()
+    public void testSaxEventToXmlElementConverterRedeclaredDefaultNamespaceUri()
             throws XMLStreamException, IOException, SAXException {
         String doubleDeclaredNamespaceUriSnippet =
                 //@formatter:off
-                  "<x xmlns:n1='foobar' xmlns='foobar'>"
-                + "    <good a='1' b='2' />"
-                + "    <good a='1' n1:a='2' />"
+                  "<x xmlns:ns1='foobar' xmlns='foobar'>"
+                + "    <good1 a='1' b='2' />"
+                + "    <good2 a='1' ns1:a='2' />"
                 + "</x>";
                 //@formatter:on
 
         String reconstructedExpectation =
                 //@formatter:off
                   "<x xmlns='foobar'>"
-                + "    <good a='1' b='2'></good>"
-                + "    <good a='1' xmlns:ns1='foobar' ns1:a='2'></good>"
+                + "    <good1 a='1' b='2'></good1>"
+                + "    <good2 a='1' xmlns:ns1='foobar' ns1:a='2'></good2>"
```

Comment:
it would, except that's not allowed. you can't have `<element oneattribute="x" oneattribute="y">` you have to namespace one of them. I put a comment on the weirdness in the `SaxEventToXmlElementConverter`